plugins/woocommerce/includes/gateways/paypal/includes/class-wc-gateway-paypal-request.php

```diff
@@ -792,15 +788,20 @@ private function get_paypal_order_items( $order ) {
 
 		foreach ( $order->get_items( array( 'line_item', 'fee' ) ) as $item ) {
 			$item_amount = $this->get_paypal_order_item_amount( $order, $item );
-			if ( 'line_item' === $item->get_type() && $item_amount < 0 ) {
-				// PayPal does not accept negative item amounts (for line items).
-				WC_Gateway_Paypal::log( sprintf( 'Order item with negative amount for PayPal order items. Order ID: %d, Item ID: %d, Amount: %f', $order->get_id(), $item->get_id(), $item_amount ), 'error' );
+			if ( $item_amount < 0 ) {
+				// PayPal does not accept negative item amounts items breakdown.
```

Comment:
Nit: this comment could be phrased better.
```suggestion
				// PayPal does not accept negative item amounts in the items breakdown, so we return an empty list.
```